dbscheme-proposal.md

```diff
@@ -0,0 +1,120 @@
+# Database Scheme
+
+## scheme
+* type (enc/sig)
+* name
+* category (lattice etc)
+* exact problems plus type of reduction (list of tuples?)
+* year (what date to use?)
+* inventors ?
+* trust in scheme (how to formalize this?)
+  * attacks
+  * cryptanalysis
+* statefulness (what options are there?)
+
+## parameter set (better name?)
+* -> scheme
+* name ("Lightsaber", "Papa Bear")
+* type/api (better name?) (ktm, kem, ...)
+* security level achieved
+  * classical (bits)
+  * quantum (bits)
+  * nist category (1..5)
+* security notion achieved (IND-CCA, EUF-CMA, ...)
+  * number of operations supported (one-time/few-time/...)
+* sizes (bytes)
+  * public key
+  * secret key (using nist api? (decaps is not given pub key))
+    * also store "real" secret key size?
```

Comment:
Maybe we should note key size and something like "optimized key size", e.g. XMSS where you could re-compute most of the key from a small key instead of storing the entire thing.

Reply:
I think we should only take the "optimized key size" because that is what needs to be transmitted or stored. Memory requirements are taken into account in the benchmark section.